clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -1272,9 +1281,23 @@ describe("Fides-js TCF", () => {
           system_legitimate_interests_preferences: { [SYSTEM_1.id]: false },
           vendor_consent_preferences: { [VENDOR_1.id]: true },
         },
-        tc_string: "CPziCYAPziCYAGXABBENATEIAACAAAAAAAAAABEAAAAA.IABE",
+        fides_string: "CPziCYAPziCYAGXABBENATEIAACAAAAAAAAAABEAAAAA.IABE",
```

Comment:
This would have been caught by typescript; should add a `FidesCookie` type declaration